lib/fedex/rate.rb

```diff
@@ -47,6 +49,8 @@ def initialize(options = {})
       @total_net_fedex_charge = (options[:total_net_fe_dex_charge] || {})[:amount]
       @total_rebates = (options[:total_rebates] || {})[:amount]
       @special_rating_applied = options[:special_rating_applied]
+      @request_xml = request_xml unless request_xml.nil?
```

Comment:
Is `nil?` the best choice here? As the default is set to empty strings `.nil?` will mean it will still be saved

Reply:
Weirdly, I wasn't able to use the rails blank? or any of those other ones, only nil?. I wanted to use present? but it threw a no method error

Reply:
This gem doesn't include any of the ActiveSupport dependencies of which things like `nil.present?` are defined in. I think we could use the safe operator here instead